src/adhocracy_core/adhocracy_core/resources/relation.py

```diff
@@ -0,0 +1,82 @@
+"""Statements about relations between process content/comments."""
+from pyramid.registry import Registry
+
+from adhocracy_core.interfaces import IItemVersion
+from adhocracy_core.interfaces import IItem
+from adhocracy_core.interfaces import IPool
+from adhocracy_core.interfaces import IServicePool
+from adhocracy_core.resources import add_resource_type_to_registry
+from adhocracy_core.resources.itemversion import itemversion_meta
+from adhocracy_core.resources.item import item_meta
+from adhocracy_core.resources.service import service_meta
+
+import adhocracy_core.sheets.relation
+import adhocracy_core.sheets.comment
+
+
+class IRelation(IItem):
+
+    """Relation versions pool."""
+
+
+class IRelationVersion(IItemVersion):
+
+    """Relation version."""
+
+
+class IPolarizationVersion(IRelationVersion):
+
+    """A polarization in a discussion."""
+
+
+polarizationversion_meta = itemversion_meta._replace(
+    content_name='PolarizationVersion',
+    iresource=IPolarizationVersion,
+    extended_sheets=(adhocracy_core.sheets.relation.IPolarization,
+                     ),
+    permission_create='edit_comment',
+)
+
+
+class IPolarization(IRelation):
+
+    """Polarization versions pool."""
+
+
+polarization_meta = item_meta._replace(
+    content_name='Polarization',
+    iresource=IPolarization,
+    element_types=(IPolarizationVersion,
+                   ),
+    item_type=IPolarizationVersion,
+    use_autonaming=True,
+    autonaming_prefix='polarization_',
+    permission_create='create_comment',
```

Comment:
We should set ìmplicit_addable to True, otherwise we cannot add the polarization to the relations service pool.